src/components/CreateVirtualFacilityModal.vue

```diff
@@ -130,6 +131,11 @@ export default defineComponent({
         this.formData.facilityId = generateInternalId(this.formData.facilityName)
       }
 
+      if(this.formData.facilityId && this.isFacilityAlreadyExists()) {
```

Comment:
I think this check is not required, as flow of code will not come here without having a facilityId.